packages/cli/src/task-runners/task-runner-process.ts

```diff
@@ -106,7 +106,7 @@ export class TaskRunnerProcess extends TypedEmitter<TaskRunnerProcessEventMap> {
 	startNode(grantToken: string, taskBrokerUri: string) {
 		const startScript = require.resolve('@n8n/task-runner/start');
 
-		return spawn('node', [startScript], {
+		return spawn('node', ['--disallow-code-generation-from-strings', startScript], {
```

Comment:
Would it be reasonable to set this up so we can unit test that this flag is being passed in?

Reply:
Sure, I'll add a unit test for that

Reply:
I wanted to add a check at runtime that this does get passed, but it's not available there